misc/formatter/modules/formatter-core/src/test/java/org/ballerinalang/formatter/core/ParserTestFormatter.java

```diff
@@ -73,6 +73,15 @@ public List<String> skipList() {
                 "qualified_identifier_assert_08.bal",
                 "conditional_expr_source_28.bal",
                 "resiliency_source_04.bal",
+                "tuple_type_source_09.bal",
+                "tuple_type_source_05.bal",
+                "tuple_type_source_08.bal",
+                "tuple_type_source_12.bal",
+                "tuple_type_source_11.bal",
+                "tuple_type_source_02.bal",
+                "tuple_type_source_10.bal",
+                "ambiguity_source_20.bal",
+                "ambiguity_source_01.bal",
```

Comment:
Any reason we are skipping these tests in the Formatter?

Reply:
they were skipped because `type T [@a x, @annot{} y]` was formatted as `type T[@ax , @annot {}y]`, I'll look at fixing the formatter

Reply:
Fixed.